.github/workflows/main.yml

```diff
@@ -0,0 +1,20 @@
+name: Main
+on: [push]
```

Comment:
can you trying adding `pull_request` here?

Reply:
`on [push, pull_request]` ?